lib/perl/Genome/Model/Tools/Somatic/FilterFalsePositives.pm

```diff
@@ -636,6 +665,10 @@ sub fails_homopolymer_check {
 sub readcount_program {
     my $self = shift;
     my $reference = $self->reference;
+    if($self->bam_readcount_version) {
+      my $bam_rc = "bam-readcount" . $self->bam_readcount_version;
```

Comment:
Yeah, you're right--that one isn't set up like most of them with a `path_for_bam_readcount_version`-like subroutine.  I woudln't worry about it!